spec/features/charge_credit_card.rb

```diff
@@ -0,0 +1,98 @@
+require 'spec_helper'
+
+feature 'Charging a credit card', { js: true } do
+  background { visit register_path }
+
+  scenario 'signup with valid user info and valid credit card' do
+    VCR.use_cassette('create_stripe_charge') do
+      fill_in 'Email Address', with: 'joesmith@gmail.com'
```

Comment:
You can refactor all of these detail steps that are repeated into methods, like the solutions do.